isis/src/control/apps/pointreg/pointreg.xml

```diff
@@ -80,6 +80,16 @@ every other measure in the point to that reference location.  An
       measures within a control point are ignored (Ignore=True), then the entire
       control point is set to ignored (Ignore=True) at the control point level.
     </p>
+
+    <p>
+      ISIS programs that must be run before running <i>pointreg</i>:
+    </p>
+    <ul>
+      <li>spiceinit</li>
+      <li>footprintint</li>
+      <li>findimageoverlaps</li>
+      <li>autoseed</li>
+    </ul>
```

Comment:
This may not necessary be true if a user is working with existing cubes that already have spice data and/or are working with an existing control networks. Maybe state that this program requires a list of cubes with SPICE data and an associated control network. Or you can reference these docs which explain from start to finish how to process a cube for bundle apps https://astrogeology.usgs.gov/docs/how-to-guides/image-processing/bundle-adjustment-in-isis/. Just not sure if that is necessary.

Reply:
Could I rephrase it, maybe say "Pointreg requires SPICE data and an associated control network, so you may need to run the following ISIS programs on your images beforehand.  See <Bundle Adjustment for more info." ... [the list]?

I saw a similar list on another program, and I thought it was really helpful because that's one of the first things I usually want to know.